Automattic/wp-calypso
# Coming Soon v2: roll back page selector

#### Changes proposed in this Pull Request

This commit reverts all of the changes from #45892, which adds a Coming Soon page selector to the page list.

We don't need the page selector feature right now but might need them later, in which case we'll re-add them.

We've also deleted the unused selectors. `isComingSoonModeActive` might be useful in the future but, for now we're not using it.

We're keeping the reducer changes since they effect the settings page cc @lsl 

#### Testing instructions

Check the testing instructions in https://github.com/Automattic/wp-calypso/pull/45892

The Coming Soon option should not be present in the page selector.

Tracking should also be gone.
